src/main/java/io/vlingo/schemata/infra/persistence/SchemataObjectStore.java

```diff
@@ -0,0 +1,102 @@
+import io.vlingo.symbio.store.dispatch.Dispatchable;
+import io.vlingo.symbio.store.dispatch.Dispatcher;
+import io.vlingo.symbio.store.object.MapQueryExpression;
+import io.vlingo.symbio.store.object.ObjectStore;
+import io.vlingo.symbio.store.object.PersistentObjectMapper;
+import io.vlingo.symbio.store.object.jdbc.jdbi.JdbiOnDatabase;
+import io.vlingo.symbio.store.object.jdbc.jdbi.JdbiOnHSQLDB;
+import io.vlingo.symbio.store.object.jdbc.jdbi.JdbiPersistMapper;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class SchemataObjectStore {
+
+    private final JdbiOnDatabase jdbi;
+    private final Map<Class<?>, PersistentObjectMapper> mappersLookup;
+
+    public SchemataObjectStore(final String username, final String password) throws Exception {
+        jdbi = initializeDatabase(username, password);
+        mappersLookup = new HashMap<>(5);
+    }
+
+    public Collection<PersistentObjectMapper> persistentMappers()  {
+        PersistentObjectMapper organizationStateMapper =
+                PersistentObjectMapper.with(
+                        OrganizationState.class,
+                        JdbiPersistMapper.with(
+                                "INSERT INTO ORGANIZATION(id, name, description) VALUES (:id, :name, :description)",
+                                "UPDATE ORGANIZATION SET name = :name, description = :description WHERE id = :id",
+                                (update,object) -> update.bindFields(object)),
+                        new OrganizationStateMapper());
+
+        mappersLookup.put(OrganizationState.class, organizationStateMapper);
+
+        return Collections.singletonList(organizationStateMapper);
+        // TODO: add more mappers
+    }
+
+    public ObjectStore objectStoreFor(
+            World world,
+            Dispatcher<Dispatchable<TextEntry, TextState>> dispatcher,
+            Collection<PersistentObjectMapper> mappers) {
+        return jdbi.objectStore(world, dispatcher, mappers);
+    }
+
+    @SuppressWarnings({ "unchecked", "rawtypes" })
+    public void register(ObjectTypeRegistry registry, ObjectStore objectStore) {
+        final Info<Organization> organizationInfo =
+                new Info(
+                    objectStore,
+                    OrganizationState.class,
+                        "vlingo-schemata",
+                    MapQueryExpression.using(Organization.class, "find", MapQueryExpression.map("id", "id")),
+                    mappersLookup.get(OrganizationState.class));
+
+        registry.register(organizationInfo);
+    }
+
+    private JdbiOnDatabase initializeDatabase(final String username, final String password) throws Exception {
+        JdbiOnDatabase jdbi = JdbiOnHSQLDB.openUsing(HSQLDBConfigurationProvider.configuration(
+                DataFormat.Native,
+                "jdbc:hsqldb:mem:",
+                "vlingo-schemata",
```

Comment:
I suggest using the name `vlingo_schemata` instead because the dash is incompatible with Postgres.

Reply:
Done, BTW do I need to load other parameters from configuration file or it is ok for now to have them hardcoded ?

Reply:
@tjaskula Ok if isolated in `Boostrap` for now. (See below comment.)